src/App.tsx

```diff
@@ -15,71 +16,106 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
-
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
-
-      {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
-
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
-
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
-
-        <tr data-cy="Good" className="has-background-success-light">
-          <td>
-            <button
-              data-cy="RemoveButton"
-              type="button"
-              className="button is-info"
-            >
-              -
-            </button>
-          </td>
-
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Jam
-          </td>
-        </tr>
-
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
-
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Garlic
-          </td>
-        </tr>
-      </tbody>
-    </table>
-  </main>
-);
+type State = {
+  selectedGood: string,
+};
+
+export class App extends React.Component<{}, State> {
+  state = {
+    selectedGood: 'Jam',
+  };
+
+  addButton = (event: React.MouseEvent<HTMLButtonElement>, good:string) => {
+    const target = event.currentTarget;
+
+    target.parentElement?.parentElement?.classList
+      .toggle('has-background-success-light');
+
+    this.setState({ selectedGood: good });
+  };
+
+  removeButton = (event: React.MouseEvent<HTMLButtonElement>) => {
+    const target = event.currentTarget;
+
+    target.parentElement?.parentElement?.classList
+      .toggle('has-background-success-light');
+
+    this.setState({ selectedGood: '' });
+  };
+
+  clearButton = () => {
+    this.setState({ selectedGood: '' });
+  };
+
+  render() {
+    const { selectedGood } = this.state;
+
+    return (
+      <main className="section container">
+        {selectedGood ? (
+          <>
```

Comment:
redundant fragment,
h1 is your wrapper

Reply:
fixed, usually have a problem with that(